### PR TITLE
ENT-2550 | update manual enrollment API to save enterprise_customer info

### PR DESCRIPTION
**Pull request by HammadAhmadWaqas**
updated manual enrollment API to save enterprise_customer info if provided.

### CODE REVIEW

**Review by ziafazal — APPROVED** [review]
👍

File: ecommerce/extensions/api/v2/views/orders.py [hunk 1]
```diff
@@ -354,11 +366,15 @@ def _get_learner_user(self, lms_user_id, learner_username, learner_email):
...
         """
-        Get or Create 100% discount offer for `Manual Enrollemnt Order`.
+        Get or Create 100% discount offer for `Manual Enrollment Order`.
         """
-        condition, __ = Condition.objects.get_or_create(proxy_class=class_path(ManualEnrollmentOrderDiscountCondition))
+        condition, __ = Condition.objects.get_or_create(
+            proxy_class=class_path(ManualEnrollmentOrderDiscountCondition),
+            enterprise_customer_name=enterprise_customer_name,
+            enterprise_customer_uuid=enterprise_customer_uuid,
+        )
```
Comment (hasnain-naveed) [inline ecommerce/extensions/api/v2/views/orders.py:378]: you should not get_or_create with `enterprise_customer_name` as it can be changed by admin panel.

Comment (HammadAhmadWaqas) [reply ecommerce/extensions/api/v2/views/orders.py:378]: @hasnain-naveed Updated.

**Review by ziafazal — APPROVED** [review]
👍

**Review by hasnain-naveed — APPROVED** [review]
👍

**Review by brittneyexline — APPROVED** [review]

**Pull request merged**